birdhouse/scripts/read-configs.include.sh

```diff
@@ -55,12 +56,18 @@ discover_compose_dir() {
             # Case of sub-subdir of sibling checkout at same level as birdhouse-deploy.
             COMPOSE_DIR="$(realpath "../../../birdhouse-deploy/birdhouse")"
         fi
-        echo "$COMPOSE_DIR"
         export COMPOSE_DIR
     fi
 }
 
 
+discover_compose_dir
```

Comment:
I think right after calling `discover_compose_dir` we need something similar to this extra block
```sh
if [ ! -f "${COMPOSE_DIR}/pavics-compose.sh" ]; then
  echo "***CRITICAL: '${COMPOSE_DIR}/pavics-compose.sh' not found, please set variable 'COMPOSE_DIR' to a valid location.  Many features depend on this variable." 1>&2
fi
```
Because this `COMPOSE_DIR` var is so critical for the platform.  If it can not be auto detected, the calling script has to be reminded to set it properly.

Reply:
Looks a bit like a chicken-and-egg situation. 
All the scripts were adjusted to use `. "${COMPOSE_DIR}/scripts/read-configs.include.sh"`.
The only way `discover_compose_dir` would be called is if `COMPOSE_DIR` was properly overridden by the caller or resolved via `pavics-compose.sh` such that `read-configs.include.sh` could be sourced. 

I don't mind adding it the explicit message to double-check and be extra sure, but I cannot see how that situation would happen.